lib/urlwatch/reporters.py

```diff
@@ -201,49 +193,96 @@ def _parts(self):
 
             yield SafeHtml('<hr>')
 
-        yield SafeHtml("""
-        <address>
-        {pkgname} {version}, {copyright}<br>
-        Website: {url}<br>
-        watched {count} URLs in {duration} seconds
-        </address>
-        </body>
-        </html>
-        """).format(pkgname=urlwatch.pkgname, version=urlwatch.__version__, copyright=urlwatch.__copyright__,
-                    url=urlwatch.__url__, count=len(self.job_states), duration=self.duration.seconds)
-
-    def _diff_to_html(self, unified_diff):
-        for line in unified_diff.splitlines():
-            if line.startswith('+'):
-                yield SafeHtml('<span class="unified_add">{line}</span>').format(line=line)
-            elif line.startswith('-'):
-                yield SafeHtml('<span class="unified_sub">{line}</span>').format(line=line)
-            else:
-                yield SafeHtml('<span class="unified_nor">{line}</span>').format(line=line)
+        yield SafeHtml(
+            '<div style="font-family:monospace;font-style:italic">\n'
+            '<span>{pkgname} {version}, {copyright}</span>\n'
+            '<address>Website: <a href="{url}">{url}</a></address>\n'
+            '<span>Watched {count} URLs in {duration} seconds</span>\n'
+            '</div>\n'
+            '</body>\n'
+            '</html>\n').format(pkgname=urlwatch.pkgname, version=urlwatch.__version__, copyright=urlwatch.__copyright__,
```

Comment:
Also here, please keep the multi-line string.

Reply:
Also here, use [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) if it is an issue. Multi-line strings are better/less noisy than this (less quotes, less `\n`).